### PR TITLE
Create standardwebpage.html

### PR DESCRIPTION
**Pull request by kanishkaa24**
I have made a standard web page.

### CODE REVIEW

**Comment by Tejas1510** [comment]
@kanishkaa24 I will go through it and let you know if any changes are required

**Comment by kanishkaa24** [comment]
@Tejas1510 
Sure

**Comment by kanishkaa24** [comment]
Screenshot (125)
Screenshot (126)
Screenshot (127)
These are the screenshots of the website I created. The Athavani and Sign In/Sign Up will lead to main website. 
@Tejas1510

**Comment by kanishkaa24** [comment]
Do let me know if there are any changes @Tejas1510.

File: standardwebpage.html [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+  <div class = "navbar">
+    <ul>
+      <li><a href="https://memoriestejastapas.netlify.app/">Sign In/ Sign Up</a></li>
+      <li><a href="#news">Contact Us</a></li>
+      <li><a href="#home">About Us</a></li>
+      <li style="float:left"><a class="active" href="https://memoriestejastapas.netlify.app/">Athavani</a></li>
+    </ul>
+  </div>
+  <div class = "image1">
+    <img src = "C:\Users\Kanishka\Downloads\Memories are forever (3).png" height="700px" width="1500px">
```
Comment (sanjay270899) [inline standardwebpage.html:81]: You have used the *absolute* path for the image, which was stored in your local Machine. I will suggest adding this image to the project folder and then use the *relative* path.

**Pull request merged**